engine/src/main/java/org/terasology/engine/core/modes/loadProcesses/CreateWorldEntity.java

```diff
@@ -31,12 +33,13 @@
 import org.terasology.engine.world.generator.WorldGenerator;
 
 import java.util.Iterator;
-import java.util.Map;
 
 /**
  */
 public class CreateWorldEntity extends SingleStepLoadProcess {
 
+    private static final Logger logger = LoggerFactory.getLogger(CreateWorldEntity.class);
+
     private final Context context;
     private final GameManifest gameManifest;
```

Comment:
would you also note that this GameManifest is a completely different game manifest from

GameManifestProvider.java

Reply:
😖